### PR TITLE
Revert "Fix migrations (#3302)"

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
This reverts commit 5dead591

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: api/pkg/crd/migrations/migrations.go [hunk 1]
```diff
@@ -57,8 +55,6 @@ func RunAll(config *rest.Config, workerName string) error {
 		return fmt.Errorf("failed to create Kuermatic client: %v", err)
 	}
 
-	config.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: scheme.Codecs}
```
Comment (alvaroaleman) [inline api/pkg/crd/migrations/migrations.go:60]: Will you create a second PR for this?

Comment (mrIncompetent) [reply api/pkg/crd/migrations/migrations.go:60]: Yes

**Pull request merged**